includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1362,12 +1362,39 @@ class_exists( 'Jetpack' ) &&
 		}
 	}
 
+	protected function wc_setup_activate_show_errors() {
+		if ( isset( $_GET['activate_error'] ) ) :
+			$error_message = $this->get_activate_error_message( $_GET['activate_error'] );
+		?>
+		<div class="woocommerce-error">
+			<p>
+				<?php echo wp_kses_post( $error_message ); ?>
+			</p>
+			<p class="wc-setup-actions">
+				<a
+					class="button"
+					href="<?php echo esc_url( $this->get_next_step_link() ); ?>">
+					<?php echo "Ok, I'll connect Jetpack later"; ?>
```

Comment:
This string needs translation.

Reply:
This was removed and the new strings are correctly translated in 48f90f3336db6e35bb8218bd558e2385e326d098